Description: Add Exec variable & Keywords to desktop file. Drop shebang.
Author: Bas Couwenberg <sebastic@debian.org>

--- a/gnome_resource/spatialite-gui.desktop
+++ b/gnome_resource/spatialite-gui.desktop
@@ -1,10 +1,10 @@
-#!/usr/bin/env xdg-open
 [Desktop Entry]
 Name=Spatialite GUI
 GenericName=GUI for Spatialite databases
 Comment=Manage Spatialite databases
-Exec=spatialite_gui
+Exec=spatialite-gui %f
 Icon=spatialite-gui
 Terminal=false
 Type=Application
 Categories=Education;Geography;Science;
+Keywords=spatial;sqlite;gis;
